.gitignore

```
@@ -0,0 +1,4 @@
+.DS_Store
+node_modules/
+.env/
```

```suggestion
env/
```
Xong rồi chạy lệnh để xóa folder trên git nha. Để folder env k được đưa lên.
`git rm -r --cached env`